rsrc/linux/com.github.Murmele.Gittyup.desktop

```
@@ -2,11 +2,11 @@
 Encoding=UTF-8
 Name=Gittyup
 Comment=Graphical Git client for Windows, Linux and macOS
-Exec=Gittyup
+Exec=/usr/bin/gittyup
```

Why specify the full path here? For distros that install `*.desktop` files it is almost always safe to assume they have the bin files in a proper location in the `$PATH`. Only in the event of a totally non-standard location (such as `/opt/blah/...` would a full path be appropriate here. In case they install to a location outside `$PATH` it would be up to the distro or installer to set this, not for an upstream project to predict.

```suggestion
Exec=gittyup
```

---

Thanks for the change.